client/src/components/Card.js

```diff
@@ -16,6 +17,66 @@ const types = {
   direct: "var(--gradient-direct)",
 };
 
+const Card = ({
+  type,
+  labels,
+  info = true,
+  settings = false,
+  start,
+  dest,
+  rider,
+  ...props
+}) => {
+  return (
+    <CardContainer type={type} {...props}>
+      <Header>
+        <Start type={type}>
+          <p>{start}</p>
+        </Start>
```

Comment:
Instead of `<div><p>{start}</p></div>`, wouldn't it be enough to directly use a `p` and remove the `div`?
Same with `Destination`.

Reply:
you are right :) should work as well... it was implemented because of the condition of the Start-Element... ill test it, as soon as possible